alpha/declcfg/declcfg_to_model_test.go

```diff
@@ -280,6 +280,155 @@ func TestConvertToModel(t *testing.T) {
 				Bundles:  []Bundle{newTestBundle("foo", "0.1.0")},
 			},
 		},
+		{
+			name:      "Error/Deprecation/UnspecifiedPackage",
```

Comment:
nit: names are for humans - machines handle whatever we throw at them. I know you're following the pattern here and you don't need to change it, but in the future we should consider using names that are English sentences